tests/CheckSshkeyTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class CheckSshkeyTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Test check_sshkey function
+     */
+    public function testCheckSshkeyValidity()
+    {
+	# Load functions
+	require_once("lib/functions.inc.php");
+
+	# Valid SSH Keys Configuration
+	$ssh_valid_key_types = array('ssh-rsa', 'ssh-dss', 'ecdsa-sha2-nistp256', 'ssh-ed25519');
+
+	# Sample inputs to test
+	$valid_key_dsa = "ssh-dss    AAAAB3NzaC1kc3MAAACBAPk0v+w59ie8OmBDlWdBKq/8YFP9h6BbnDxUkwLNlUQ27sCdI3Ko6R/ivEC22Q+SKSF+vL8TThfRWlwhDuro0aqK/0oqwz0d8wyxwWKSlygzME7PoAA5xIxAwTSF3apxGhMkZ3xVQyVVO0P/w4nRGOYEEfmeZdR6NMtZej9UszmfAAAAFQC8QxCoKLq+0e23E50YpO+gEHKrqwAAAIAGvgIwBXV7SxhhdudjJKBpjSVc+SOM5oxMT/Yx8qfkx6nGrvsmBMtCtjkDYsYxfC/GAJK2LEvfPUah5a0U33EmfIXxSjc+MGOd90i69FKSfaa5nuGb+9T/bhevYDsJvBkvl+rWABLCnDXvqUY0ZJ5JO6mT9xD+mq7Q7hy6kZzVAgAAAIARAuLXJVf3tMsASLxm++efey9dQl+8EsveT08E45eKxJU0F5yIdqkHcuWquDP9xI9e35cKx13W5uSGIr9TDH9wNjyY0aX3MkUeYM9UQut3bHr/tH4TjM87hDitOCuRERGZwvvhHX3mxzug0N44DLyXmZFvV2Wq+MInLdVujcricg== a@b";
+	$valid_key_ecdsa = "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBCt5Z3IMcaL0OAQ62QpGLvRE+5GsolzqT7gGQ/76gz348IJfb+6MyXPQbT3BmNLtSw88UaaKp0HOQFj6f3US8rU= a@b";
```

Comment:
Can you please add ecdsa-sha2-nistp384 and ecdsa-sha2-nistp521

Reply:
ok